client/client.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2015, 2016 Eris Industries (UK) Ltd.
+// This file is part of Eris-RT
+
+// Eris-RT is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Eris-RT is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Eris-RT.  If not, see <http://www.gnu.org/licenses/>.
+
+package client
+
+import (
+	"fmt"
+
+	"github.com/tendermint/go-rpc/client"
+
+	"github.com/eris-ltd/eris-db/account"
+	tendermint_client "github.com/eris-ltd/eris-db/rpc/tendermint/client"
+	"github.com/eris-ltd/eris-db/txs"
+)
+
+type NodeClient interface {
```

Comment:
I agree names are important, but I do like `Node` for the reason I call it here `ErisNodeClient` and `ErisKeyClient`; and yes `TendermintNode` (it is a node on its own at that consensus level);

`Node` in `ErisNodeClient` is a running instance of eris-db